sos/report/plugins/ceph_mgr.py

```diff
@@ -103,17 +85,67 @@ def setup(self):
             "version"
         ]
 
+        directory = ''
```

Comment:
This declaration / assignment is redundant:

    >>> if 1 < 3:
    ...   directory = 'first'
    ... else:
    ...   directory = 'second'
    ... 
    >>> directory
    'first'
    >>>

Reply:
The thought was that I needed directory declared outside the ifs to use it later with the values the if conditions set it to.